Resources/config/form-type.xml

```diff
@@ -9,5 +9,13 @@
             <tag name="form.type" alias="cmf_core_checkbox_url_label" />
             <argument type="service" id="router" />
         </service>
+
+        <service id="cmf_core.form.extension.publishable" abstract="true" class="Symfony\Cmf\Bundle\CoreBundle\Form\Extension\PublishableExtension">
+            <!-- <argument /> Extended type fqcn -->
```

Comment:
When extending abstract services, arguments are appended to the parent's one. So in this case we will have argument 0 to be empty and argument 1 to be the fact.